slayer_rails.gemspec

```diff
@@ -21,10 +21,12 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_runtime_dependency('slayer', '~> 0.2')
+  spec.add_runtime_dependency 'slayer', '~> 0.2'
+  spec.add_runtime_dependency 'rails', '>= 4.2.0'
```

Comment:
Should we put an upper-cap on this? Should we have `['>= 4.2.0', < '6.0']`?

Also, is 4.2.0 truly our lower bound, or can we support 4.1 and 4.0?